portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,122 @@
+                            <a href="https://penntoday.upenn.edu/news/smart-aviary-poised-break-new-ground-behavioral-research">research</a> for a computational
+                            neuroscience lab at Penn.
+                        </p>
+                    </div>
+
+                    <h2>AUTOMATIC ROOM ALLOCATOR</h2>
+                    <h3>for Ronald McDonald House Philadelphia &nbsp; • &nbsp;<a class="light_blue">Flask, PostgreSQL</a></h3>
+                    <!-- add later
+                    &nbsp; • &nbsp; <a><i class="fab fa-github"></a></i> -->
+                    <div class="box">
+                        <p>
+                            Room allocator for Ronald McDonald House client purposed to reduce manual entry and processing time of room requests. 
+                            Worked on UI/UX design, clientel relations, and full-stack development.
+                        </p>
+                    </div>
+
+                    <h2>PHILADELPHIA CONTRACT DATABASE</h2>
+                    <h3>for the Philadelphia City Controller &nbsp; • &nbsp;<a class="light_blue">Flask, PostgreSQL</a></h3>
+                    <div class="box">
+                        <p>
+                            Searchable database for the City of Philadelphia containing all city, lobbyist, and campaign finance contracts. Worked as 
+                            project manager, involving project structure breakdown and task allocation, full-stack development, clientel relations, 
+                            and UI/UX design.
+                        </p>
+                    </div>
+
+                    <h2>ANIMAL VOCALIZATION GENERATIVE NETWORK</h2>
+                    <h3>for Passerine Cowbirds &nbsp; • &nbsp;<a class="light_blue">Pandas, Tensorflow, Sci-Kit Learn</a></h3>
+                    <div class="box">
+                        <p>
+                            Generative neural network autoencoder using MDS and variational models purposed to automatically visualize male passerine songbird songbird
+                            in order to perform analysis between mating posture and birdsong. Worked on autoencoder implementation.
+                        </p>
+                    </div>
+
+                    <h2>ANTI-RESUME</h2>
+                    <h3>for The Signal Penn &nbsp; • &nbsp;<a class="light_blue">React, Node, MongoDB</a></h3>
+                    <div class="box">
+                        <p>
+                            Web application showcasing the "anti-resumes" of Penn seniors and alumni, to highlight both failures and accomplishments of people at 
+                            Penn that wouldn’t normally show up on a resume, and to promote the message that individuals are more than their resume, GPA, internship, 
+                            or job. Worked on full-stack development.
```

Comment:
This is super cool! Any chance you have something you can link if the web-app still exists and is publicly accessible?

Reply:
!! We're working on deployment/final bug fixes but it should be up in a week or so. I'll make sure to link it :))